src/governance/classes.h

```diff
@@ -117,7 +117,7 @@ class CSuperblock : public CGovernanceObject
     uint256 nGovObjHash;
 
     int nBlockHeight;
-    int nStatus;
+    SeenObjectState nStatus;
```

Comment:
mixing `State` and `Status`. should either do `s/SeenObjectState/SeenObjectStatus/` or `s/Status/State/` in `nStatus`/`GetStatus`/`SetStatus` etc imo.